jas/secrets/secretsscanner.go

```diff
@@ -25,61 +26,49 @@ const (
 type SecretsScanType string
 
 type SecretScanManager struct {
-	secretsScannerResults []*sarif.Run
-	scanner               *jas.JasScanner
-	scanType              SecretsScanType
-	configFileName        string
-	resultsFileName       string
+	secretsScannerVulnerabilitiesResults []*sarif.Run
```

Comment:
why did you leave the []*sarif.Run inside SecretScanManager and did not remove it like in the other managers (iac, sast). I cannot see any other usage for this field besides creating it in newSecretsScanManager

Reply:
missed that, removed